Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -105,7 +96,7 @@ Units: this MUST be in kilowatt hours (kWh).
 
 #### Location-Based Marginal Carbon Intensity (`I`)
 
-The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this standard uses a consequential approach, marginal emissions rates MUST be used for electricity consumption. 
+The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this standard uses a impact-based approach, marginal emissions rates MUST be used for electricity consumption. 
```

Comment:
HDR: Removing to separate discussion.
```suggestion
The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this standard uses a consequential approach, marginal emissions rates MUST be used for electricity consumption. 
```